polaris-pipeline/Common/Exceptions/DdeiClientException.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+
+namespace Common.Exceptions;
+
```

Comment:
Exception is not marked as Serializable. Missing serialization constructor definition.